samples/snippets/csharp/VS_Snippets_ADO.NET/DataWorks ConnectionStringSettings.RetrieveFromConfigByProvider/CS/source.cs

```diff
@@ -32,11 +29,10 @@ static string GetConnectionStringByProvider(string providerName)
             foreach (ConnectionStringSettings cs in settings)
             {
                 if (cs.ProviderName == providerName)
-                    returnValue = cs.ConnectionString;
-                break;
+                    return cs.ConnectionString;
```

Comment:
Our style would allow one of two forms here. Either add `{` `}` around the `return` statement, or put the `return` on the same line as the `if`:

```csharp
if (cs.ProviderName == providerName) return cs.ConnectionString;
```

Option 2:

```csharp
if (cs.ProviderName == providerName)
{
   return cs.ConnectionString;
}
```